javamelody-core/src/main/java/net/bull/javamelody/internal/publish/Statsd.java

```diff
@@ -77,12 +77,15 @@ static Statsd getInstance(String contextPath, String hostName) {
 				address = statsdAddress;
 				port = DEFAULT_STATSD_PORT;
 			}
+
+			String statsdPrefix = Parameter.STATSD_PREFIX.getValueOrDefault("javamelody.${context}.${host}.");
 			// contextPath est du genre "/testapp"
 			// hostName est du genre "www.host.com"
-			final String prefix = "javamelody." + contextPath.replace("/", "") + '.' + hostName
-					+ '.';
+			statsdPrefix = statsdPrefix.replace("${context}", contextPath.replace("/", "")).
+					replace("${host}", hostName);
```

Comment:
I was not thinking of something dynamic like that. Given that the other parameters are not dynamic, Can we remove those `replace` and just have the default prefix?

Reply:
It currently supports what you requested above. How can we achieve that without any placeholders and `replace`?

> Perhaps other users may want to customize the prefix for statsd, instead of shortening it. For example, they may want to include some 'app' context without hosts or some 'host' context such as 'host1' and 'host2' without app.

Reply:
You probably mean that if someone needs to have only 'app' or only 'host' it will be put in the configuration like 'javamelody.mynewapplication.' or 'javamelody.myawsinstance.', right? Shall I simplify it then?

Reply:
Yes, that's it. For the sake of the behavior of the parameters, this one like the others, yes I think so.